### ISSUE TITLE
Backdrop container

### ISSUE THREAD

**Issue by btecu** [sorich87/bootstrap-tour#428]
I'm trying to select the specific element with `element` but have the backdrop container (light area) on the parent (outer) element. I thought `backdropContainer` would do that, but apparently it doesn't. Is there an alternative for this or is `backdropContainer` supposed to be used for this case?

**Comment by jsmits21** [comment]
I want to do the same thing. I might contribute to the project in the near future.

A good example is my case. I have an application navbar. My first step tells you this is the navbar ect. I then want to dive into the navbar and tell you about some functionality within it, like the expand hamburger button for example. It's kind of odd to just highlight the button when showing something within the navbar. It would be cool to have an additional option for the step's backdrop to keep the full navbar highlighted. 

Basically, a "backdropElement" option that overrides the step's "element", allowing a parent item to be highlighted when discussing a child.

**Comment by jsmits21** [comment]
Also, it sounds easy enough to code providing you know the right place in the code. If you guys point me to some line numbers I can probably knock this out for ya :)

**Comment by btecu** [comment]
By looking at the source code, it looks like it `backdropContainer` doesn't do what we need.
Here is where the backdrop class is being added (always to the element) https://github.com/sorich87/bootstrap-tour/blob/c77e80f74a2885ed9fb90972a41eb26d111ebd36/build/js/bootstrap-tour.js#L811 so it should be easy to either change `backdropContainer` or have another option take that backdrop class instead.

(I really dislike `coffee script` - so good luck if you plan on contributing to a `coffee script` project.)

**Comment by rhertogh** [comment]
@btecu Is it the same problem as PaulStevensLM describes here: GH-343?

**Comment by btecu** [comment]
@rhertogh I'm not sure, I think it may be a different issue.

**Comment by codylaird** [comment]
I'm having the same issue with backdropContainer, or misunderstanding of what it's meant to be used for.

**Comment by sorich87** [comment]
There's a new backdropElement option that does that.

**Comment by Kiruwagaka** [comment]
@sorich87 I don't see that option anywhere in the documentation http://bootstraptour.com/api/ , could you point me to it?
